models/ac_generator.h

```diff
@@ -116,6 +134,24 @@ class ac_generator : public Node
 
   // ------------------------------------------------------------
 
+  // These friend declarations must be precisely here.
```

Comment:
Also this comment arrived from another model, where I put it in 2009. I am not sure why I wrote it then, I believe, looking at other models, that the declarations could also be placed above the `class Parameter {};` declaration. `iaf_psc_alpha.h` provides a more sensible comment:
```c++
// The next two classes need to be friends to access the State_ class/member
```